### PR TITLE
Moved database tables and associated models to model project

### PR DESCRIPTION
**Pull request by githubmo**
### What

- Deleted code that is now in model and using model.
- No more schema creation and no more creation of tables.
- No more spring datasource creation.
- Using EntityManager and named queries in the model.
- Using the `job_creator` user instead of the `data_discovery` user in our databases.

### How to review

clean build, run it, make sure it works and all endpoints are correct.

### Who can review

Anyone but mo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ As per the stub, this offers two API calls:
 
 ### Create Job
 
-POST to /job with payload:
+POST to /jobDto with payload:
```
Comment (YetAnotherMatt) [inline README.md]: Find/replace error?

File: README.md [hunk 2]
```diff
@@ -62,14 +62,14 @@ You will receive back a response like:
 }
 ```
 
-### Check job status
+### Check job statusDto
```
Comment (YetAnotherMatt) [inline README.md]: Find/replace error

File: README.md [hunk 3]
```diff
@@ -62,14 +62,14 @@ You will receive back a response like:
...
 
 Take the `id` from the response you got when creating the job and perform a GET on `/job/{id}` to get a response like:
 
 ```json
 {
   "id": "3fdb1aed-123f-4258-b76b-5620190c7524",
-  "status": "Complete",
+  "statusDto": "Complete",
```
Comment (YetAnotherMatt) [inline README.md]: Should this still be status?

**Review by YetAnotherMatt — CHANGES_REQUESTED** [review]

**Comment by githubmo** [comment]
Now returning 400 for when a dataset doesn't exist. Example:

```
{
  "timestamp": 1487757437201,
  "status": 400,
  "error": "Bad Request",
  "exception": "uk.co.onsdigital.job.exception.NoSuchDataSetException",
  "message": "Dataset not found: 5a07f00a-34b1-45ed-ba5e-e582cdc5a59c",
  "path": "/job"
}
```

**Review by YetAnotherMatt — APPROVED** [review]

**Pull request merged**